ios/PhylloConnectModule.swift

```diff
@@ -57,22 +58,78 @@ public class PhylloConnectModule: RCTEventEmitter {
     }
     
 
-    func getEnvironment(env:String) -> PhylloEnvironment {
-        switch env {
-        case "development":
-            return PhylloEnvironment.dev
-        case "sandbox":
-            return PhylloEnvironment.sandbox
-        case "production":
-            return PhylloEnvironment.prod
-        default:
-            return PhylloEnvironment.sandbox
-        }
-    }
+
```

Comment:
Delete the commented code (unless this is temporary removal and we will need the same code again in future)